### ISSUE TITLE
Design for the inclusion of leadership PACs on sponsoring candidate profile pages

### ISSUE THREAD

**Issue by JonellaCulmer** [fecgov/fec-cms#4253]
**What we're after:**
We've recently added functionality to search for leadership PAC sponsors on the committee datatable as well as added the sponsor name to leadership PAC "about this committee" on committee profile pages. 

We need to also explore including leadership PAC information on candidate profiles pages for those candidates/officeholders who have sponsored them. 

The main concern for this was recently explained succinctly by press: 

> "...technically leadership PACs are not authorized committees, nor are they legally affiliated with any authorized committee of the candidate (a technicality that’s in the statute  itself). So to be the most precise, we may not want to group leadership PACs under authorized committees, since they operate under different rules and contribution limits."

**Considerations:**
- [ ] How might we link the candidate and committee pages together? What happens when a candidate runs for a different seat?

### Related issues
- [ ] https://github.com/fecgov/fec-cms/issues/2369#issuecomment-426312206 Meet and identify data issues to connect sponsoring candidates and leadership PACs

### Completion criteria
- [ ] Finalized design which adds leadership PACs to sponsoring candidate/officeholder profile pages
- [ ] Followup implementation ticket has been created

**Comment by JonellaCulmer** [comment]
@PaulClark2 @AmyKort @patphongs With the addition of candidate ID for leadership PAC sponsors, we were able to add sponsor information to the committee search and committee profile pages. On committee profile pages, we do not link to the candidate's profile page. For those candidates with multiple candidate profile pages, like Kamala Harris, we can only see the name in one place. For her it's her Senate page, not presidential. 

We are able to do the same thing with the committee information on candidate profile pages - display committee information but not link it to the committee profile page. But we should be prepared to explain, if asked, why a committee is available on one candidate profile page and not another. This is just a subset of the candidate profile pages, but will occur for anyone who runs for a different seat. Because of the somewhat large number of instances, we may want to explain that limitation ahead of this feature's implementation. 

We can also consider linking to the respective candidate and committee profile pages, but the problem will still remain that some and not all pages will be linked together.

**Comment by JonellaCulmer** [comment]
@PaulClark2 @AmyKort @patphongs Here are two preliminary options for displaying sponsored leadership PAC committees. These were made under the assumption that we will link to the PACs' committee profile page. That is still TBD based on my previous comment. 

### Option 1
This is if we want to include all committees together in one place under the header Committees. Each committee type is broken out into sections but nested under the same header. 
<img width="386" alt="Screen Shot 2020-12-16 at 10 21 13 AM" src="https://user-images.githubusercontent.com/31663028/102368689-1fb5a700-3f89-11eb-9707-bc449d8d9a04.png">


### Option 2
This is if we want to break out the leadership PAC from under other authorized committees. This would further reinforce that the two committee are not affiliated if we feel that extra separation is necessary. We can also consider adding additional explanatory language to this section if we feel the need to elaborate on why they are apart, but I'm not sure that's necessary at this time.
<img width="387" alt="Screen Shot 2020-12-16 at 10 21 20 AM" src="https://user-images.githubusercontent.com/31663028/102368703-23492e00-3f89-11eb-9724-4b0743c04c24.png">

**Comment by PaulClark2** [comment]
Thank you @JonellaCulmer. I don't have a strong opinion about this, but I like Option 1 better for some reason. It seems more compact and a bit "cleaner" to look at, if that makes any sense.

**Comment by PaulClark2** [comment]
I think the section titles (campaign committees, JFs and leadership PACs) of Option 1 make the committee's relationship to the candidate clear. We could add glossary links after each section title/committee type if folks think that'd help users understand the relationship to the candidate better.

**Comment by JonellaCulmer** [comment]
@dorothyyeager @kathycarothers Please see Paul's comment and provide your thoughts if you have a preference.

I also wanted to get your thoughts on the labels. Does "Sponsored leadership political action committees" or "Sponsored committees" and "Leadership political action committees" work for you?

**Comment by kathycarothers** [comment]
@JonellaCulmer @dorothyyeager I think either "Sponsored leadership political action committees" or "Leadership political action committees" works.  I think either way we need to keep leadership in the wording.

**Comment by dorothyyeager** [comment]
@JonellaCulmer @kathycarothers I agree with Kathy. The people we regulate and the people who research them say "Leadership PAC" so I think we should go with "Sponsored leadership PACs" and link the glossary definition of leadership PAC. (I don't even think we need to spell out political action committee as we don't in the glossary or the [regs](https://www.fec.gov/regulations/100-5/2020-annual-100#100-5-e-6).

**Comment by patphongs** [comment]
@JonellaCulmer @PaulClark2 I like option 1 as well. I wonder how clear it is though that the only money that is in the candidate's financial totals are the from their authorized committees only? We only seem to say which committees are part of their financial totals in the first financial summary tab. The more committees we add could cause confusion about this and may be good to have some explanatory text.

![Screen Shot 2020-12-17 at 2 42 57 PM](https://user-images.githubusercontent.com/12799132/102535411-30901680-4076-11eb-89d3-be65c7dc0129.png)

**Comment by JonellaCulmer** [comment]
@patphongs @PaulClark2 @dorothyyeager @kathycarothers 
Thank you all for your feedback. It seems that the consensus is to go with option one and to modify the text to say "Leadership PACs" and include a glossary link. The glossary link should be able to assist with any questions about the PAC and its non-relation to the other committees. 

Perhaps I can run this by Myles who expressed the same concern we had about including leadership PACs with other authorized committees and see if he has any concerns. 

**If we're all in agreement, please give me a thumbs up on this comment.**

**Comment by JonellaCulmer** [comment]
Updated mockup based on previous comment. I'm uncertain about the placement of the glossary icon for the "authorized campaign committee" section. I tried including the icon after the word "authorized" but it didn't work quite as well at the end of the sentence. However there is no term in the glossary for "authorized campaign committee." There is authorized committee and principal campaign committee. I'm assuming we link that section to authorized committee?

<img width="779" alt="Screen Shot 2020-12-17 at 4 05 36 PM" src="https://user-images.githubusercontent.com/31663028/102543793-d09f6d00-4081-11eb-9d9d-cdfc96888492.png">
